Buildfile

```diff
@@ -7,6 +7,6 @@ cookbook.depends 'propsd' do |propsd|
   propsd.path './cookbook'
 end
 
-profile :default do |default|
-  default.chef.run_list 'propsd::default'
+profile :default do |test|
```

Comment:
That's fine but, the default profile doesn't give you an instance that even starts.  What is the default profile for if not for local testing?